.changelog/1879.breaking.md

```diff
@@ -0,0 +1 @@
+Rename to ROSE Wallet
```

Comment:
- Rename to Oasis ROSE Wallet
- why it's breaking? we do not plan to bump major version with this update, right?

Reply:
Im not sure. Process then?

Reply:
I would classify it as a feature and not breaking.

We will bump major version when we release extension and web from the same codebase.